frontend/src/components/VideoCardPage/VideoCardPage.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
Found another bug. If I delete all the videos from recycling bin, go to user profile page and click the arrow icon back in top bar, then all the deleted videos reappear again. It has possibly to do something with react router history prop and it's goBack function